src/components/CertificateDropZone/Views/UnverifiedView.test.js

```diff
@@ -38,58 +37,151 @@ const VALID_VERIFICATION_STATUS = {
   }
 };
 
-it("renders correctly when the hash mismatch", () => {
-  const tree = renderer
-    .create(
+describe("UnverifiedView", () => {
+  it("displays hash error if the hash is invalid", () => {
+    const wrapper = shallow(
       <UnverifiedView
+        handleRenderOverwrite={() => {}}
         verificationStatus={{
           ...VALID_VERIFICATION_STATUS,
           hash: { checksumMatch: false }
         }}
+        resetData={() => {}}
       />
-    )
-    .toJSON();
-  expect(tree).toMatchSnapshot();
-});
+    );
+    const errorContainerElm = wrapper
+      .find("DetailedErrors")
+      .dive()
+      .find("#error-tab0");
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.HASH].failureTitle
+    );
+    expect(errorContainerElm.text()).toContain(
+      MESSAGES[TYPES.HASH].failureMessage
+    );
+  });
 
-it("renders correctly when the certificate is not issued", () => {
-  const tree = renderer
-    .create(
+  it("displays error if the document is not issued", () => {
```

Comment:
Suggestion => it displays issuing error ...

Reply:
fixed